styles/modules/_pageNav.scss

```diff
@@ -106,37 +106,75 @@
 
   .searchWrapper {
     .addressBarIndicators {
-      top: 0px;
-      right: 0px;
       position: absolute;
-      width: 110px;
+      top: 0;
+      right: 0;
       height: 100%;
+      display: flex;
+      align-items: center;
+      padding: 1px 5px 1px 0;
+      box-sizing: border-box;
+
+      &:before {
+        content: '';
+        width: 15px;
+        height: 100%;
+        background: linear-gradient(to right, transparent, $primary 50%);
```

Comment:
Any styling that deals with color classes should be made in _themes.scss. If we ever implement user driven theming, it will all be driven by that file.

Just to clarify, it's fine if the vast majority of the css lives here, just any rules that change color should be moved to that file.